### ISSUE TITLE
Redshift closes when connecting a monitor

### ISSUE THREAD

**Issue by tomrozb** [jonls/redshift#302]
I've a problem with redshift that it sometimes closes after I connect a monitor using HDMI cable.

I wanted to attach some logs but I couldn't find any. Where does redshift store logs or crash information?

**Comment by maandree** [comment]
There are no logs. But there is some output to stderr.

**Comment by tomrozb** [comment]
Is it possible to capture the standard error when redshift is started from autostart? I mean it's added to .config/autostart

**Comment by jonls** [comment]
@tomrozb Thanks for the report. `redshift-gtk` will probably not produce any relevant error messages. Is it possible for you to run redshift (not redshift-gtk) from a terminal? That should allow you to see any error messages.

**Comment by tomrozb** [comment]
Yes it is. The only logs after a few days of using it and seeing it crashing from time to time are presented below:

```
Started Geoclue provider `Geoclue Master'.
According to the geoclue provider we're at: 47.62, -122.14
```

As you can see nothing special, just some geo logs. How can I debug what's wrong and why it's crashing?
